src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Pbkdf2Implementation.Android.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal partial class Pbkdf2Implementation
+    {
+        public static unsafe void Fill(
+            ReadOnlySpan<byte> password,
+            ReadOnlySpan<byte> salt,
+            int iterations,
+            HashAlgorithmName hashAlgorithmName,
+            Span<byte> destination)
+        {
+            Debug.Assert(!destination.IsEmpty);
+            Debug.Assert(hashAlgorithmName.Name is not null);
+            // Fall back to managed implementation since Android doesn't support the full Pbkdf2 APIs
+            // until API level 26.
+            using Rfc2898DeriveBytes deriveBytes = new Rfc2898DeriveBytes(
```

Comment:
Please use the braced using statement.

To my knowledge, we don't have any braceless usings in product code.

Reply:
Well...

https://github.com/dotnet/runtime/blob/ecc354a61c909a1960af7e5a73eb9a0d255c564e/src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CertificateAssetDownloader.cs#L240

Reply:
Touche.

I still prefer the braces :)